apps/site/pages/docs/api-extensions.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: '    Overview'
```

Comment:
```suggestion
title: 'Overview'
```

Reply:
It was! For some reason (that I haven't investigated yet), when we remove these spaces, Nextra arranges the documents alphabetically in the left menu of the Portal. Removing these spaces ensures that the overview will become the first document in the section. In the video below, In the example with the doc "Starting a new FastStore project", note that when I remove the spaces, the "Dashboard" doc takes the place of the "Starting a new FastStore project".


https://github.com/vtex/faststore/assets/67270558/ca0585b0-f789-4cf7-a715-2e70af85b66b

Reply:
Changes updated [here](https://github.com/vtex/faststore/pull/1942/commits/0e4bbc260ba06f301f0adac1426d8e2b84af7996).